torch/utils/benchmark/utils/valgrind_wrapper/timer_interface.py

```diff
@@ -599,6 +600,7 @@ def run(args: List[str], **kwargs: Any) -> Tuple[CompletedProcessType, str]:
                     args,
                     stdout=f_stdout_stderr,
                     stderr=subprocess.STDOUT,
+                    check=True,
```

Comment:
Error code is checked on line 655, isn't it, and when author want it to check the code, it calls it with `check=True` kwarg on line 672
```suggestion
                    check=False,
```